pkg/wipeout/tasks.go

```diff
@@ -1,133 +0,0 @@
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package wipeout
-
-import (
-	"context"
-
-	"github.com/apigee/registry/log"
-	"github.com/apigee/registry/pkg/connection"
-	"github.com/apigee/registry/rpc"
-)
-
-// DeleteApiTask deletes a specified API.
-type DeleteApiTask struct {
-	client connection.RegistryClient
-	name   string
-}
-
-func NewDeleteApiTask(client connection.RegistryClient, name string) *DeleteApiTask {
-	return &DeleteApiTask{
-		client: client,
-		name:   name,
-	}
-}
-
-func (task *DeleteApiTask) String() string {
-	return "delete " + task.name
-}
-
-func (task *DeleteApiTask) Run(ctx context.Context) error {
-	log.Debugf(ctx, "Deleting %s", task.name)
-	return task.client.DeleteApi(ctx, &rpc.DeleteApiRequest{Name: task.name})
-}
-
-// DeleteVersionTask deletes a specified version.
-type DeleteVersionTask struct {
-	client connection.RegistryClient
-	name   string
-}
-
-func NewDeleteVersionTask(client connection.RegistryClient, name string) *DeleteVersionTask {
-	return &DeleteVersionTask{
-		client: client,
-		name:   name,
-	}
-}
-
-func (task *DeleteVersionTask) String() string {
-	return "delete " + task.name
-}
-
-func (task *DeleteVersionTask) Run(ctx context.Context) error {
-	log.Debugf(ctx, "Deleting %s", task.name)
-	return task.client.DeleteApiVersion(ctx, &rpc.DeleteApiVersionRequest{Name: task.name})
-}
-
-// DeleteSpecTask deletes a specified spec.
-type DeleteSpecTask struct {
-	client connection.RegistryClient
-	name   string
-}
-
-func NewDeleteSpecTask(client connection.RegistryClient, name string) *DeleteSpecTask {
-	return &DeleteSpecTask{
-		client: client,
-		name:   name,
-	}
-}
-
-func (task *DeleteSpecTask) String() string {
-	return "delete " + task.name
-}
-
-func (task *DeleteSpecTask) Run(ctx context.Context) error {
-	log.Debugf(ctx, "Deleting %s", task.name)
-	return task.client.DeleteApiSpec(ctx, &rpc.DeleteApiSpecRequest{Name: task.name})
-}
-
-// DeleteDeploymentTask deletes a specified deployment.
-type DeleteDeploymentTask struct {
-	client connection.RegistryClient
-	name   string
-}
-
-func NewDeleteDeploymentTask(client connection.RegistryClient, name string) *DeleteDeploymentTask {
-	return &DeleteDeploymentTask{
-		client: client,
-		name:   name,
-	}
-}
-
-func (task *DeleteDeploymentTask) String() string {
-	return "delete " + task.name
-}
-
-func (task *DeleteDeploymentTask) Run(ctx context.Context) error {
-	log.Debugf(ctx, "Deleting %s", task.name)
-	return task.client.DeleteApiDeployment(ctx, &rpc.DeleteApiDeploymentRequest{Name: task.name})
-}
-
-// DeleteArtifactTask deletes a specified artifact.
-type DeleteArtifactTask struct {
```

Comment:
Since we also need to keep this task, maybe we should keep this file?

Reply:
Sure